duckduckgo/Android
# Feature/remove tab ux experiment

Task/Issue URL: https://app.asana.com/0/488551667048375/1145180114989609
Tech Design URL: 
CC: 

**Description**:
Removes the old Tab implementation and the experimentation for the new tab UX

**Steps to test this PR**:
1. Test both a clean install, and upgrading from `develop` (from the old Tab UX)
1. Open some tabs, verify the new Tab UX screen is shown, the previews generate as expected


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
